Lesson05/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <script src="./script.js" charset="utf-8"></script>
+    <title>Lesson 05</title>
+  </head>
+  <body>
+    <h1>Bank of Belford</h1>
+    <button onclick="bankApp('Enter a letter to begin banking:\nQ - to quit application\nW - to withdraw funds\nD - to deposit funds\nB - to view balance')" type="button" name="button">Start Banking</button>
```

Comment:
I'd rather this long string lives in your JS file, it's a bit long and clutters up the HTML.

Reply:
Noted- I'll be sure to keep my html clutter-free